ci/code_checks.sh

```diff
@@ -316,6 +316,11 @@ fi
 ### DOCSTRINGS ###
 if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
 
+
+    MSG='Validate docstrings (D201, D202, D204, D207, D208, D209, D213, D300, D409, D411, D412, D414)' ; echo $MSG
+    pydocstyle pandas --select=D201,D202,D204,D207,D208,D209,D213,D300,D409,D411,D412,D414
```

Comment:
Would it make sense to have the codes in `setup.cfg`? I guess we could then call `pydocstyle pandas` locally and get this validation.